apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -747,17 +747,15 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         if (!tabPaneExtensions && applicationConfig.editApplication.extendTabs
             && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC
             && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
-            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_SAML
-            && application?.templateId !== ApplicationManagementConstants.MOBILE) {
```

Comment:
Won't it affect On-Prem IS?

Reply:
We don't have quickstart guides for applications in On-Prem IS. So we are adding the mobile app quickstart also to Asgardeo only.